src/MockNet/Http/Internal/MockHttpClientException.cs

```diff
@@ -11,80 +13,108 @@ public class MockHttpClientException : Exception
     {
         internal static async Task<MockHttpClientException> NoSetupAsync(SystemHttpRequestMessage request)
         {
-            return new MockHttpClientException(
-                string.Format(
-                    CultureInfo.CurrentCulture,
-                    @"{0} request failed.\n{1}",
-                    await Utils.HttpRequestMessage.ToStringAsync(request),
-                    "All requests on the mock must have a corresponding setup."));
+            var message = new StringBuilder()
+                .AppendLine()
+                .AppendLine()
+                .AppendLine("Missing setup for:")
+                .Append(await Utils.HttpRequestMessage.ToStringAsync(request))
+                .AppendLine();
```

Comment:
Is the `StringBuilder` necessary here? It seems a string interpolation would work:

```csharp
var requestAsString = await Utils.HttpRequestMessage.ToStringAsync(request);
var message = $"\n\nMissing setup for:\n{requestAsString}\n";
```

Reply:
Similar for all the methods below.

Reply:
Updated.